docs/index.md

```diff
@@ -1 +1,62 @@
-# Aurelian
\ No newline at end of file
+# Aurelian
+
+## Getting Started
+
+### Prerequisites
+
+Ensure you have the following installed:
+
+- [Poetry](https://python-poetry.org/docs/#installation) (for managing dependencies)
+- Python 3.11 or later
+
+### Installation
+
+1. **Clone the repository:**
+   ```sh
+   git clone https://github.com/your-org/aurelian.git
+   cd aurelian
+   ```
+2. **Install dependencies:**
+   ```sh
+   poetry install
+   ```
+3. **Verify installation:**
+   ```sh
+   poetry run aurelian --help
+   ```
+
+### Authentication Setup
+
+Aurelian uses Logfire for logging, which requires authentication. If you encounter the following error:
```

Comment:
Explain in a few words what logfire is, this seems very much more than a simple logging mechanism.

Reply:
done, thanks Nico